docs/training-debug/index.txt

```diff
@@ -1,343 +1,395 @@
-   is for ``--local --test`` mode. If you get errors related to ``unmarshaling JSON`` when trying to
-   submit the experiment to the cluster, that is an indication that the experiment config has
-   errors. Please review the :ref:`experiment configuration <experiment-configuration>`.
+   .. code:: yaml
 
-Again, if you are unable to identify the root cause of the issue yourself, please do not hesitate to
-contact Determined through our `community support
+      resources:
+      slots_per_trial: 1
+
+#. Create an experiment with the ``--test`` argument, omitting the ``--local`` argument:
+
+   .. code:: bash
+
+      det experiment create myconfig.yaml my_model_dir --test
+
+#. Perform failure diagnostics:
+
+   If you can run local test mode inside a notebook or shell but are unable to successfully submit
+   an experiment, make sure that notebook or shell customizations you might have made are replicated
+   in your :ref:`experiment configuration <experiment-configuration>`, such as:
+
+   -  If required, a custom Docker image is set in the experiment configuration.
+
+   -  ``pip install`` or ``apt install`` commands needed in the interactive environment are built
+      into a custom Docker image or included in the ``startup-hook.sh`` file in the model definition
+      directory root. See :ref:`startup-hooks` for more information.
+
+   -  Custom bind mounts required in the interactive environment are specified in the experiment
+      configuration.
+
+   -  Environment variables are correctly set in the experiment configuration.
+
+   If no customizations are missing, the following, new layers introduced with a cluster-managed
+   experiment could be the problem cause:
+
+   -  The ``checkpoint_storage`` settings are used for cluster-managed training. If
+      ``checkpoint_storage`` is not configured in the experiment configuration or the master
+      configuration, an error message can occur during experiment configuration validation before
+      the experiment or trials are created. Correct this by providing a ``checkpoint_storage``
+      configuration in one of the following locations:
+
+      -  :ref:`master-configuration`
+      -  :ref:`experiment-configuration`
+
+   -  For a cluster-based experiment, configured ``checkpoint_storage`` settings are validated
+      before training starts. The message ``Checkpoint storage validation failed``, indicates that
+      you should review the ``checkpoint_storage`` setting values.
+
+   -  The experiment configuration is more strictly validated for cluster-managed experiments than
+      for ``--local --test`` mode. Errors related to ``unmarshaling JSON`` when attempting to submit
+      the experiment to the cluster indicate that the experiment configuration has errors. Review
+      the :ref:`experiment configuration <experiment-configuration>`.
+
+If you are unable to identify the root cause of the issue, contact Determined `community support
```

Comment:
suggest reprasing "the root cause of the issue" to match the phrasing X used before in "If no customizations are missing, the following, new layers introduced with a cluster-managed experiment could be the X"

Reply:
Changed according to what I think you meant.

Reply:
What I mean is: a couple of sentences earlier you say 

> If no customizations are missing, the following new layers introduced with a cluster-managed experiment could be the cause of the problem:"

I'm suggesting changing what you had before: 

> If you are unable to identify ~the root cause of the issue~, ...

to

> If you are unable to identify **the cause of the problem**, ...

specifically to match the phrasing you used just before.

I'm not a big fan of the newest phrasing:

> If no customizations are missing and the new layers introduced with a cluster-managed experiment are not the cause of the problem, contact Determined

because I would always rather encourage users to try to figure it out themselves before coming to us, and also because a user might find it very difficult to determine if "the new layers introduced with a cluster-managed experiment are not the cause of the problem"

Reply:
Agree. Fixed.